corehq/apps/hqwebapp/views.py

```diff
@@ -1290,7 +1290,7 @@ def log_email_event(request, secret):
     # https://docs.aws.amazon.com/ses/latest/DeveloperGuide/event-publishing-retrieving-sns-examples.html
 
     if secret != settings.SNS_EMAIL_EVENT_SECRET:
-        return HttpResponse(status=403)
+        return HttpResponse(f"Incorrect secret: {secret}", status=403, content_type='text/plain')
```

Comment:
@proteusvacuum bump

Reply:
I'm not exactly sure what the threat model is here, since it is just sending back verbatim what is sent in the URL, but I've removed it anyway.
https://github.com/dimagi/commcare-hq/pull/27929